cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -1392,4 +1392,268 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
     cy.wait('@refreshCellQuery')
     cy.wait('@refreshSecondQuery')
   })
+
+  // auto refresh tests
+
+  describe('Dashboard auto refresh', () => {
+    let routeToReturnTo = ''
+    const jumpAheadTime = (timeAhead = '00:00:00') => {
+      return Cypress.moment()
+        .add(Cypress.moment.duration(timeAhead))
+        .format('YYYY-MM-DD HH:mm:ss')
+    }
+
+    beforeEach(() => {
+      cy.get('@org').then(({id: orgID, name}: Organization) => {
+        cy.createDashboard(orgID).then(({body}) => {
+          cy.fixture('routes').then(({orgs}) => {
+            routeToReturnTo = `${orgs}/${orgID}/dashboards/${body.id}`
+            cy.visit(routeToReturnTo)
+            cy.getByTestID('tree-nav')
+          })
+        })
+        cy.window().then(win => {
+          cy.wait(1000)
+          win.influx.set('newAutoRefresh', true)
```

Comment:
nit: I know this is convention, can we add a TODO to remove the wait for when the feature flag is removed, just in case we forget?

Reply:
done.